benchmarks/benchmarks/GRO.py

```diff
@@ -2,12 +2,20 @@
 
 import numpy as np
 from MDAnalysis.coordinates.GRO import GROReader
+from MDAnalysis.topology.GROParser import GROParser
 from MDAnalysisTests.datafiles import GRO
+import MDAnalysis as mda
 
 class GROReadBench(object):
 
-    def time_read_GRO_file(self):
-        """Benchmark reading of standard test
-        suite GRO file.
-        """
+    def time_read_GRO_coordinates(self):
+        """Benchmark reading of standard testsuite GRO file."""
         GROReader(GRO)
+
+    def time_parse_GRO_file(self):
+        with GROParser(GRO) as p:
```

Comment:
are you ok with having this timing report the context manager initialization on top of the `parse()` method?